.github/workflows/apk_check.yml

```diff
@@ -0,0 +1,17 @@
+# name: Apk check
+
+# # on pr on main branch
+# on:
+#   pull_request:
+#     branches:
+#       - main
+
+# jobs:
```

Comment:
we gonna reput this one day ?

Reply:
If one day, we decide to add apk check in ci, we can reuse this file soooo